README.md

```diff
@@ -229,32 +229,26 @@ Notice that we have to pass the option `-s` to `pytest` so that it shows what we
 The maze is a grid. Each square in the grid is defined by its coordinates. The default width of the maze is `32` squares, the default height is `16` squares. The coordinate system has the origin `(0, 0)` in the top left (North-West) of the maze and its maximum value `(31, 15)` in the bottom right (South-East). Each square which is not a wall can be empty or contain a food pellet or one or more bots. The different mazes are called `layouts`. For the tournament all layouts will have the default values for width and height and will have a wall on all squares around the border.
 
 ### The `move` function
-**`move(bot, state) ⟶ (x, y), state`**
+**`move(bot, state) ⟶ (x, y)`**
 
 The `move` function gets two input arguments:
 
 - **`bot`** is a reference to the bot in your team corresponding to the current turn. It is an instance of the [`Bot` object](#the-bot-object), which contains all information about the current state of the game
 
-- **`state`** is an arbitrary object, `None` by default, which can be used to hold state between rounds. Example of usage for `state` can be found in [demo04_basic_attacker.py](demo04_basic_attacker.py), [demo05_basic_defender.py](demo05_basic_defender.py), [demo06_one_and_one.py](demo06_one_and_one.py):
+- **`state`** is a place which can be used to hold state between rounds. It is initialized as an empty dictionary when the game starts, and the `move` function can store whatever it wants in it.
```

Comment:
maybe `place` ⟶ `variable`?

Reply:
I don't like "variable", because there are no variables in python. OTOH, "object" would be more correct, but confusing. I think "place" is a good middle ground.

Reply:
> I don't like "variable", because there are no variables in python. OTOH, "object" would be more correct, but confusing. I think "place" is a good middle ground.

I know there are no "variables" in Python, but this is the name people, and our students, give to these things. By calling it "place" you are inventing a term that I have never heard at ASPP (nor anywhere else, to be honest). It can be called "object", if in the docs it is clear how to use it and that that "object" is actually a dictionary.
But again, people call this things "variables" or "parameters" and we are not making a course about correct programming terminology. Is a bit like the discussion about threading, hyper-threading, processes and threads, the more you try to be precise, the less people will follow you.

Reply:
Right, but in this particular case, the distinction between the variable and the object it points to is esp. important. E.g. saying `state = { 'foo':'bar' }` will not have the desired effect. But OK, I'll change it to variable. I don't think it's particularly important.